oc_art_to_ftp/app/art_to_ftp.py

```diff
@@ -0,0 +1,318 @@
+import io
+import logging
+import os
+import posixpath
+
+from oc_cdtapi import NexusAPI
+from ftplib import FTP, error_perm
+
+
+class ArtToFTP:
+
+    def __init__(self):
+        """
+        initialize class
+        """
+        self.supported_media = ['artifactory', 'ftp']
+
+    def gav_copy(self, gav, target_path):
+        """
+        copies specified gav from artifactory to target_path on FTP
+        """
+        logging.debug('Reached gav_copy')
+        logging.debug('gav: [%s]' % gav)
+        logging.debug('target_path: [%s]' % target_path)
+        
+        if not posixpath.isabs(target_path):
+            logging.debug('Target path is not absolute, adding leading slash')
+            target_path = '/' + target_path
+            logging.debug('target_path: [%s]' % target_path)
+        
+        logging.debug('Initializing NexusAPI')
+        na = NexusAPI.NexusAPI()
+        logging.debug('Checking existence of [%s]' % gav)
+        
+        if not na.exists(gav):
+            logging.error('Source artifact was not found')
+            return self._response(404, 'Source artifact not found')
+        
+        logging.debug('Artifact exisits, downloading')
+        data = na.cat(gav, binary=True)
```

Comment:
Seems not good idea to sotre binary artifact data in the memory by the way like this.
Suggestion is to write it to temporary file.
```
import tempfile
_data = tempfile.TemporaryFile()
na.cat(gav, binary=True, write_to=_data)
...
# to get file size:
_data.flush()
_size_in_bytes = os.fstat(_data.fileno()).st_size
```